.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: python
+sudo: required
+dist: xenial
```

Comment:
to get python 3.7 support - 
https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905